src/app/components/PageLayout/Header.tsx

```diff
@@ -49,7 +66,7 @@ export const Header: FC = () => {
               showText={!scrollTrigger && !isMobile}
             />
           </Grid>
-          {scope && (
+          {scope && needsScopeSelector() && (
```

Comment:
I would avoid doing function calls, isn't this static? In a sense that it wouldn't change on each re-render?